run_service.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+# coding=utf-8
+
+import time
+import BaseHTTPServer
+import urllib
+import json
+import base64
+import ConfigParser
+cf = ConfigParser.ConfigParser()
+
+cf.read('./src/conf/spartan.config')
+
+HOST_NAME = str(cf.get("base", "host"))
+PORT_NUMBER = int(cf.get("base", "port"))
+
+CONIFRM_PATH = './log'
+
+class HttpHandler(BaseHTTPServer.BaseHTTPRequestHandler):
```

Comment:
The entire flow is confusing for me, is there any flow design doc?, feel free to leave TODO here